src/parser/character/index.js

```diff
@@ -22,54 +22,47 @@ let getProficiencies = data => {
 
 let get5EBuiltIn = data => {
   let results = {
-    "powerfulBuild": false,
-    "savageAttacks": false,
-    "elvenAccuracy": false,
-    "halflingLucky": false,
-    "initiativeAdv": false,
-    "initiativeAlert": false,
-    "jackOfAllTrades": false,
-    "weaponCriticalThreshold": 20,
-    "observantFeat": false,
-    "remarkableAthlete": false,
-    "reliableTalent": false,
+    powerfulBuild: false,
+    savageAttacks: false,
+    elvenAccuracy: false,
+    halflingLucky: false,
+    initiativeAdv: false,
+    initiativeAlert: false,
+    initiativeHalfProf: false,

```

Comment:
```suggestion
    jackOfAllTrades: false,
```
Needs to be `jackOfAllTrades`